### PR TITLE
Add information about risks associated with giving super user access to a user

### PR DESCRIPTION
**Pull request by michalkleiner**
### Description:

Add information about risks associated with giving super user access to a user. There's a difference in the copy based on what plugins the instance has enabled.

Ref. DEV-18465

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Review by caddoo — DISMISSED** [review]

File: plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_superuser_confirm.png
Comment (sgiehl) [inline plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_superuser_confirm.png:1]: This change looks wrong

Comment (michalkleiner) [reply plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_superuser_confirm.png:1]: I think that is caused by the UI component being longer with the new text, so the modal is displayed lower.

Comment (sgiehl) [reply plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_superuser_confirm.png:1]: It should only capture the modal, the position should actually be irrelevant 🤔 
There are also some user tests in UsersManagement failing, which might be caused by the newly added tests. Seems a step to go back to the overview in the end is missing

Comment (michalkleiner) [reply plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_superuser_confirm.png:1]: I don't see any UsersManagement tests failing in the last build https://builds-artifacts.matomo.org/matomo-org/matomo/dev-18465/10826869143/, do you have any more details, please?

Comment (sgiehl) [reply plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_superuser_confirm.png:1]: You need to look at the github action logs and not solely at the build artifacts: https://github.com/matomo-org/matomo/actions/runs/10826869143/job/30038766749?pr=22554#step:3:1221

Comment (michalkleiner) [reply plugins/UsersManager/tests/UI/expected-screenshots/UsersManager_superuser_confirm.png:1]: @sgiehl the confirmation screenshot has been reverted and the relevant failing UI tests resolved.

**Review by sgiehl — APPROVED** [review]

**Pull request merged**